src/elements/StatusBadge/StatusBadge.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Status Badge
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import Common from '../Common/Common';
+
+const StatusBadge = ({ children, active, paused, pending, archived, ...other }) => {
```

Comment:
You're not using `children` anywhere, so it can be omitted from the destructuring.